test/integration/guest_user_test.rb

```diff
@@ -29,7 +29,7 @@ def setup
 
   test 'a guest user can view home page' do
     visit root_path
-    assert page.has_content?('Coffee House')
+    assert page.has_content?('Cinema Coffee')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.